.gitignore

```diff
@@ -7,4 +7,4 @@
 .nyc_output/
 .DS_Store
 .vscode
-*config.json
\ No newline at end of file
+testing_config.json
```

Comment:
Nah, I want the example checked in but the actual super seekrit file ignored.

Same pattern as `example.env` and `.env`